pkgs/development/python-modules/sentencepiece/default.nix

```
@@ -6,10 +6,13 @@
 
 buildPythonPackage rec {
   pname = "sentencepiece";
-  inherit (sentencepiece) version src meta;
+  inherit (sentencepiece) version src;
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ sentencepiece ];
+  buildInputs = [ sentencepiece.dev ];
 
   sourceRoot = "source/python";
+
+  # sentencepiece installs 'bin' output.
+  meta = builtins.removeAttrs sentencepiece.meta [ "outputsToInstall" ];
```

I think this could be reverted with your newest changes, its not in the other expression anymore
```suggestion
  meta = builtins.removeAttrs sentencepiece.meta [ "outputsToInstall" ];
```

---

If I remove that and inherit `meta`, `meta.outputsToInstall` has an incorrect value of `[ "bin" ]` again.

---

oh, I didn't realize multiple outputs did this... hmm